components/NewRealmWizard/components/steps/AddNFTCollectionForm.tsx

```diff
@@ -73,36 +90,64 @@ async function enrichCollectionInfo(connection, collectionKey) {
   )
 }
 
-async function getNFTCollectionInfo(connection, collectionKey) {
-  const { data: result } = await deprecated.Metadata.findByMint(
-    connection,
-    collectionKey
-  )
-  console.log('NFT findByMint result', result)
-  if (result?.collection?.verified && result.collection?.key) {
+async function getNFTCollectionInfo(
+  connection: Connection,
+  collectionKey: string
+) {
+  const result = await fetch('https://graph.holaplex.com/v1', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      query: getNftQuery(1, 0),
+      variables: {
+        collection: new PublicKey(collectionKey),
+      },
+    }),
+  })
+  const body = await result.json()
+  const mintToCheck = body.data?.nfts[0]?.mintAddress
+  const metaplex = new Metaplex(connection)
+  const metaplexData = await metaplex.nfts().findByMint({
+    mintAddress: new PublicKey(mintToCheck),
+  })
+
+  console.log('NFT findByMint result', metaplexData)
+  if (metaplexData?.collection?.verified && metaplexData.collection?.address) {
     // here we were given a child of the collection (hence the "collection" property is present)
     const collectionInfo = await enrichCollectionInfo(
       connection,
-      result.collection.key
+      metaplexData.collection.address.toBase58()
     )
-    const nft = await enrichItemInfo(result.data, result.data.uri)
+    const nft = await enrichItemInfo(metaplexData, metaplexData.uri)
     collectionInfo.nfts = [nft]
     return collectionInfo
   } else {
     // assume we've been given the collection address already, so we need to go find it's children
-    const children = await deprecated.Metadata.findMany(connection, {
-      updateAuthority: result!.updateAuthority,
+    const result = await fetch('https://graph.holaplex.com/v1', {
```

Comment:
may use `axios.post` it reduces the noise around `method`, `headers` etc.

I think something like:

```
const result = await axios.post(
    'https://graph.holaplex.com/v1',
    {
        query: getNftQuery(1000, 0),
        variables: {
          collection: new PublicKey(collectionKey),
        },
     },
  )
```